src/Microsoft.DotNet.Interactive/Utility/StringExtensions.cs

```diff
@@ -33,7 +33,7 @@ internal static class StringExtensions
         }
         else if (lines.Length > 1)
         {
-            firstLine = firstLine + " ...";
+            // firstLine = firstLine + " ...";
```

Comment:
Ah, I didn't mean to revert it. The logic looked weird so I was checking if it's actually needed.

Reply:
This is just appending ellipsis if the content happens to span multiple lines (since we are trimming out all lines except the first one). I think it should be preserved.

Reply:
I'll replace this in my next PR.